services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks.py

```diff
@@ -57,6 +58,26 @@ async def _retry_docker_compose_start(
     return await docker_compose_start(compose_spec, settings)
 
 
+@retry(
+    wait=wait_random_exponential(),
+    stop=stop_after_delay(5 * _MINUTE),
+    retry=retry_if_result(lambda result: result is False),
+    reraise=False,
```

Comment:
if this does not raise when it did not work, why do you still try to start them?

Reply:
I tend to agree with SAN as well on thisone.

When retrial fails, you should raise error so the caller can react. The way you have now is that it will continue silently ...

Reply:
That's a slip-up. Now raising an error if failed to start!